lib/reporters/xunit.js

```diff
@@ -130,7 +130,7 @@ XUnit.prototype.test = function(test) {
 
   if (test.state === 'failed') {
     var err = test.err;
-    this.write(tag('testcase', attrs, false, tag('failure', {}, false, cdata(escape(err.message) + '\n' + err.stack))));
```

Comment:
Wouldn't it be enough to stop using `escape` on `err.message`, since `escape` is being used in `cdata` already?

Removing the `CDATA` tag is not a change that would go into this major version, for the record.